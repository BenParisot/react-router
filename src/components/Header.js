import React from 'react';
import { 
  Link
} from 'react-router-dom';

//this needs to be a class that has state
export default function Header() {
  //state should be user color choice
  return (
    <header>
      <nav>
        <Link to="/red"></Link>
        <Link to="/blue">Blue</Link>
        <Link to="/yellow">Yellow</Link>
        <Link to="/:color">Pick A Color</Link>
      </nav>
    </header>
  );
}
