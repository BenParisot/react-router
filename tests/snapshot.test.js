import React from 'react';
import { shallow } from 'enzyme';
import Header from '../src/components/Header';

describe('Header component', () => {
  it('displays the header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
