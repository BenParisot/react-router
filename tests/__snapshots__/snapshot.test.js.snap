// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component displays the header 1`] = `
<header>
  <nav>
    <Link
      to="/red"
    />
    <Link
      to="/blue"
    >
      Blue
    </Link>
    <Link
      to="/yellow"
    >
      Yellow
    </Link>
    <Link
      to="/:color"
    >
      Pick A Color
    </Link>
  </nav>
</header>
`;
